src/drivers/cmakeFileApiDriver.ts

```diff
@@ -421,11 +421,26 @@ export class CMakeFileApiDriver extends CMakeDriver {
     }
 
     get executableTargets(): ExecutableTarget[] {
-        return this.uniqueTargets.filter(t => t.type === 'rich' && (t as RichTarget).targetType === 'EXECUTABLE')
-            .map(t => ({
-                name: t.name,
-                path: (t as RichTarget).filepath
-            }));
+        const uniqueExecTargets = this.uniqueTargets.filter(t => t.type === 'rich' && (t as RichTarget).targetType === 'EXECUTABLE');
+        const executableTargetsWithInstall = uniqueExecTargets.map(t => ({
+            name: t.name,
+            path: (t as RichTarget).filepath,
+            isInstallTarget: false
+        }));
+
+        for (const t of uniqueExecTargets) {
+            const target = t as RichTarget;
+            if (target.installPaths && target.installPaths.length > 0) {
+                for (const installPath of target.installPaths) {
+                    executableTargetsWithInstall.push({
+                        name: target.name + ' (' + installPath.subPath + ')',
```

Comment:
I tend to think we should append "Install" somewhere, similar to VS, is there a reason we can't do that?